src/crossfilter.js

```diff
@@ -1328,6 +1329,21 @@ function crossfilter() {
     return data;
   }
 
+  // Returns row data with all dimension filters applied
+  function allFiltered() {
+    var array = [],
+        i = 0;
+
+      while (i < n) {
```

Comment:
Generally the Crossfilter code base uses for-loops. Can you switch over to `for(i=0;i<n;i++)` to align?